features/zoeken/met-burgerservicenummer/overzicht.feature

```diff
@@ -0,0 +1,38 @@
+# language: nl
+Functionaliteit: gegevens raadplegen van een persoon
+
+  Als consumer
+  wil ik de actuele gegevens van één of meer personen kunnen raadplegen door het opgeven van zijn/hun burgerservicenummer
+
+    Achtergrond:
+      Gegeven de persoon 'Robin'
+      En de persoon 'Saskia'
+      En de persoon 'Gerda'
+
+  Regel: een persoon moet worden gezocht met zijn burgerservicenummer om zijn actuele gegevens te kunnen raadplegen
+
+    Scenario: het burgerservicenummer van één persoon is opgegeven
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met burgerservicenummer van 'Robin'
+      Dan wordt 'Robin' gevonden
+
+    Scenario: het burgerservicenummer van meer personen is opgegeven
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met burgerservicenummer van 'Robin, Saskia en Gerda'
+      Dan worden 'Robin, Saskia en Gerda' gevonden
+
```

Comment:
er is bij raadplegen geen regel over overleden personen, in tegenstelling tot de zoekoperaties. dan moet hier impliciet uit blijken dat ook overleden personen kunnen worden geraadpleegd. @CathyDingemanse s het niet duidelijker om dit als voorbeeld op te nemen (een aparte regel lijkt me niet nodig)

Reply:
zie overleden-personen-zoeken.feature